### PR TITLE
Add version command and update documentation

### PR DESCRIPTION
**Pull request by SunSince90**
This pull requests updates the CNWAN Reader with a `version` command that prints the current version of the reader.

This will help when providing help to users, so that we know what version they are running.

Additionally, documentation is updated with dependencies and a way to know how to run this as a binary.

This pull requests fixes and/or addresses #8 and #10.

### CODE REVIEW

**Review by arnatal — COMMENTED** [review]
Thanks Elis. Code changes for adding the version look very good. 

Looking at the updated README I have a comment. I wonder if the sections should maybe be:

Overview
Service Registries
Getting the code
Installation (native and docker)
Usage (native and docker)
OpenAPI
Contributing
License

The reason is that some of the "Before Install" only applies if running the reader natively and similarly the "Usage" section needs to be adapted when running in docker. For instance, current "Service Directory" section and "Mount Credentials" section are kind of the same except that one is for native and the other is for docker. They can be explained together just saying one is for native and the other is for Docker.

What do you think?

**Comment by SunSince90** [comment]
> Thanks Elis. Code changes for adding the version look very good.
> 
> Looking at the updated README I have a comment. I wonder if the sections should maybe be:
> 
> Overview
> Service Registries
> Getting the code
> Installation (native and docker)
> Usage (native and docker)
> OpenAPI
> Contributing
> License
> 
> The reason is that some of the "Before Install" only applies if running the reader natively and similarly the "Usage" section needs to be adapted when running in docker. For instance, current "Service Directory" section and "Mount Credentials" section are kind of the same except that one is for native and the other is for docker. They can be explained together just saying one is for native and the other is for Docker.
> 
> What do you think?

Good point. I have rewritten basically the full Readme to make it more understandable. Let me know!

File: README.md [hunk 1]
```diff
@@ -102,133 +244,97 @@ Please follow [OpenAPI Specification](#openapi-specification) to learn more
...
-Please remember that the `-v` flag must come *before* the name of the
-image.
+Now you can use all other flags as specified in [Usage](#usage) and,
+specifically, you can use `--service-account` as
+`--service-account /credentials/serv-acc.json`.
 
-All other arguments are the same as described in [Usage](#usage).
+Read the next section for a full Docker example.
 
 ### Docker Example
```
Comment (arnatal) [inline README.md:339]:
```suggestion
#### Docker Example
```

File: README.md [hunk 2]
```diff
@@ -3,21 +3,28 @@
...
-* [Installing](#installation)
+* [Installation](#installation)
+  * [Go Get](#go-get)
+  * [Clone the Project](#clone-the-project)
+  * [Run as a Docker Container](#run-as-a-docker-container)
 * [Usage](#usage)
   * [CNWAN Adaptor](#cnwan-adaptor)
+  * [Metadata Key](#metadata-key)
   * [Service Directory](#service-directory)
   * [Example](#example)
```
Comment (arnatal) [inline README.md:23]:
```suggestion
  * [Binary Example](#binary-example)
```

**Review by arnatal — APPROVED** [review]
Thanks Elis, the new README looks fantastic!

File: README.md (hunk 1 above)
Comment (SunSince90) [reply README.md]: I have amended and re-pushed this. Thank you for spotting it!

File: README.md (hunk 2 above)
Comment (SunSince90) [reply README.md]: I have amended and re-pushed this. Thank you for spotting it!

**Pull request merged**